NEWS.md

```diff
@@ -140,6 +140,12 @@ Reference tab. (#2174)
 - The meaning of `date_imputation = "mid"` was clarified in the documentation of
 the imputation functions, e.g., `derive_vars_dtm()`. (#2222)
 
+- Added an example derivation of `DTHCGR1` to the ADSL vignette. (#2218)
+
+## Templates
```

Comment:
Could we just put this in the Various section and not make a new section in our changelog and maintain consistency??

Reply:
Thanks, actioned.